src/Parser/VectorTileParser.js

```diff
@@ -0,0 +1,85 @@
+import Protobuf from 'pbf';
+import { VectorTile } from '@mapbox/vector-tile';
+import GeoJsonParser from './GeoJsonParser';
+
+function readPBF(file, options) {
+    const vectorTile = new VectorTile(new Protobuf(file));
+
+    const layers = Object.keys(vectorTile.layers);
+
+    if (layers.length < 1) return;
+
+    // We need to create a featureCollection as VectorTile does no support it
+    const geojson = {
+        type: 'FeatureCollection',
+        features: [],
+        crs: { type: 'EPSG', properties: { code: 4326 } },
+        extent: options.extent,
+    };
+
+    layers.forEach((layer_id) => {
+        const l = vectorTile.layers[layer_id];
+
+        for (let i = 0; i < l.length; i++) {
+            let feature;
+            // We need to move from TMS to Google/Bing/OSM coordinates
+            // https://alastaira.wordpress.com/2011/07/06/converting-tms-tile-coordinates-to-googlebingosm-tile-coordinates/
+            // Only if the layer.origin is top
+            if (options.origin == 'top') {
+                feature = l.feature(i).toGeoJSON(options.coords.col, options.coords.row, options.coords.zoom);
+            } else {
+                const y = 1 << options.coords.zoom;
+                feature = l.feature(i).toGeoJSON(options.coords.col, y - options.coords.row - 1, options.coords.zoom);
+            }
+            if (layers.length > 1) {
+                feature.properties.vt_layer = layer_id;
+            }
+
+            geojson.features.push(feature);
+        }
+    });
+
+    return GeoJsonParser.parse(geojson, {
+        json: true,
```

Comment:
the `json` option is not used anymore, please remove it ([it is now automatic](https://github.com/iTowns/itowns/blob/master/src/Parser/GeoJsonParser.js#L273-L275))

Reply:
Thanks !